cli/azd/pkg/auth/cache.go

```diff
@@ -36,6 +48,35 @@ func (a *msalCacheAdapter) Replace(ctx context.Context, cache cache.Unmarshaler,
 		return err
 	}
 
+	// In msal v1.0, keys were stored with mixed casing; in v1.1., it was changed to lower case.
+	// This handles upgrades where we have a v1.0 cache, and we need to convert it to v1.1,
+	// by normalizing the appropriate key entries.
+	c := map[string]json.RawMessage{}
+	if err = json.Unmarshal(val, &c); err == nil {
+		for _, contractKey := range contractFields {
+			if _, ok := c[contractKey]; ok {
```

Comment:
nit. rename `ok` to something that better explains what's ok there.  Probably `foundKey`

Reply:
changed to `found`